app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationURLBar.java

```diff
@@ -92,7 +92,10 @@ public boolean onEditorAction(TextView aTextView, int actionId, KeyEvent event)
     public void setURL(String aURL) {
         int index = -1;
         if (aURL != null) {
-            index = aURL.indexOf("://");
+            if (aURL.startsWith("jar:") || aURL.startsWith("resource:"))
```

Comment:
Could this cause intermittent blank URLS while loading a web. I mean you have a correct URL, and while a jar:// URL is received it will show blank but a shirt time before showing the final URL. 

I think a single URL bar transition would be better from a design perspective (from previous  URL to next final URL, without the intermediate blank)

Reply:
I couldn't reproduce the intermediate jar: URL issue but I've separated it in two cases as we always want to set the URL blank for resource: URL.